src/schema/ecommerce/orders.ts

```diff
@@ -84,6 +97,10 @@ export const Orders = {
       mode,
       state,
       sort,
+      first,
+      after,
+      last,
+      before,
```

Comment:
We talked about trying to be clever with something like `Object.keys( pageable({}) )` but that seemed like overkill.